opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -415,8 +395,10 @@ def create_span(
                 kind=kind,
                 links=links,
             )
-
-        return trace_api.DefaultSpan(context=context)
+            span.start(start_time=start_time)
+        else:
+            span = trace_api.DefaultSpan(context=context)
```

Comment:
Would love some feedback on whether or not this is the right thing to do here, since a DefaultSpan no longer has a `start()` method, I'm just returning it after creation

Reply:
I think this is fine, a DefaultSpan doesn't even know whether it was started anyway.